shared/chat/conversation/info-panel/container.tsx

```diff
@@ -113,6 +115,18 @@ const ConnectedInfoPanel = Container.connect(
       dispatch(Chat2Gen.createLoadAttachmentView({conversationIDKey, fromMsgID, viewType})),
     _onMediaClick: (message: Types.MessageAttachment) =>
       dispatch(Chat2Gen.createAttachmentPreviewSelect({message})),
+    _onShowBlockConversationDialog: (others: Array<string>, team: string) => {
+      dispatch(
+        RouteTreeGen.createNavigateAppend({
+          path: [
+            {
+              props: {blockByDefault: true, convID: conversationIDKey, others, team},
```

Comment:
did we decide to show `others` for teams?

Reply:
I thought so. It seems very relevant in this particular case (existing team) because someone in that team in particular is the source of spam or harassment.

Reply:
OK I made Y2K-1113 to virtualize this since it can be big.